src/php/ext/grpc/channel_credentials.c

```diff
@@ -140,10 +144,24 @@ PHP_METHOD(ChannelCredentials, createSsl) {
                          "createSsl expects 3 optional strings", 1 TSRMLS_CC);
     return;
   }
+
+  php_grpc_int hashkey_len = root_certs_length + cert_chain_length;
+  char hashkey[hashkey_len];
+  if (root_certs_length > 0) {
+    strcpy(hashkey, pem_root_certs);
+  }
+  if (cert_chain_length > 0) {
+    strcpy(hashkey, pem_key_cert_pair.cert_chain);
+  }
+
+  char *hashstr = emalloc(41);
```

Comment:
It's hidden in each object's PHP's destructor: https://github.com/grpc/grpc/blob/master/src/php/ext/grpc/php7_wrapper.h#L56